src/wrappers/DappProvider/CustomComponents.tsx

```diff
@@ -21,50 +18,32 @@ export interface CustomComponentsType {
     component: typeof TransactionsTracker;
     props?: TransactionsTrackerType;
   };
-  batchTransactionsSender?: {
-    component: typeof BatchTransactionsSender;
-    props?: object;
-  };
   batchTransactionsTracker?: {
     component: typeof BatchTransactionsTracker;
     props?: BatchTransactionsTrackerProps;
   };
 }
 
 export function CustomComponents({
-  customComponents,
-  enableBatchTransactions
+  customComponents
 }: {
   customComponents?: CustomComponentsType;
-  enableBatchTransactions?: boolean;
 }) {
   const transactionSender = customComponents?.transactionSender;
   const transactionTracker = customComponents?.transactionTracker;
-  const batchTransactionsSender = customComponents?.batchTransactionsSender;
   const batchTransactionsTracker = customComponents?.batchTransactionsTracker;
 
   const TxSender = transactionSender?.component ?? TransactionSender;
   const TxTracker = transactionTracker?.component ?? TransactionsTracker;
-  const BatchTxsSender =
-    batchTransactionsSender?.component ?? BatchTransactionsSender;
   const BatchTxsTracker =
     batchTransactionsTracker?.component ?? BatchTransactionsTracker;
 
   return (
     <>
-      {!Boolean(enableBatchTransactions) && (
-        <>
-          <TxSender {...transactionSender?.props} />
-          <TxTracker {...transactionTracker?.props} />
-        </>
-      )}
+      <TxSender {...transactionSender?.props} />
+      <TxTracker {...transactionTracker?.props} />
 
-      {Boolean(enableBatchTransactions) && (
-        <>
-          <BatchTxsSender {...batchTransactionsSender?.props} />
-          <BatchTxsTracker {...batchTransactionsTracker?.props} />
-        </>
-      )}
+      <BatchTxsTracker {...batchTransactionsTracker?.props} />
```

Comment:
I think this should be integrated by default in TxTracker

Reply:
For the moment we can't. The props necessary for these components are totally different. We can't mix them.